perception/autoware_lidar_centerpoint/include/autoware/lidar_centerpoint/postprocess/non_maximum_suppression.hpp

```diff
@@ -28,36 +28,12 @@ namespace autoware::lidar_centerpoint
 {
 using autoware_perception_msgs::msg::DetectedObject;
 
-// TODO(yukke42): now only support IoU_BEV
-enum class NMS_TYPE {
-  IoU_BEV
-  // IoU_3D
-  // Distance_2D
-  // Distance_3D
-};
-
```

Comment:
Why do we remove this in this PR? I kind of understand the benefit of this modification, but even so, would be better to do it in another PR.

Reply:
The logic checks the NMS type everytime, but this logic do not support multiple NMS models.
Even you have plan to implement other mode, this should come at that moment.

Reply:
This PR is to revise the NMS logic and improve the performance. Under this concept, removing unused parameters and logic is also included.

Reply:
OK, understood. (Also the description update LGTM)